src/app/features/management/user-management-view/create-user-view/create-user-view.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="view" [appShowLoadingOverlay]="creatingUser.loadingChange() | async">
+  <h1 i18n>Create User</h1>
+  <form (submit)="createUser()">
+    <mat-form-field class="app-edit-input">
+      <mat-label i18n>Username</mat-label>
+      <input matInput [formControl]="form.controls.username">
+      <mat-error *ngIf="form.controls.username.errors && form.controls.username.errors['required']" i18n>Username is
+        required.
+      </mat-error>
```

Comment:
see comments for edit-view. please move content to separate line. applies to all other errors in this file as well.